0.download_data/scripts/nbconverted/download_data.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
There is actually no need for this script. You can load all training data instances directly from the URL (as you've done here in line 28) in all downstream modules. Does this make sense?

This being said, as you have it here is totally fine (maybe even preferable in some scenarios)

Reply:
This does make sense